packages/block-library/src/post-title/block.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "core/post-title",
+	"category": "common",
+	"attributes": {
+		"title": {
+			"type": "string",
+			"source": "post",
+			"attribute": "title"
```

Comment:
I might think to call this `property`, particularly if we just consider the post an object from which we're picking property values. Also avoids ambiguity on block attributes.

Reply:
I went with attributes because all the selectors use "post attributes" as the terminology.

Reply:
> I went with attributes because all the selectors use "post attributes" as the terminology.

Hmm, that's fair. In retrospect I think they might have been more canonically referred to as properties of the post, but I'm fine targeting consistency if even for consistency's sake.

Reply:
Yeah I like "properties" more too.